src/settings.js

```diff
@@ -197,38 +197,48 @@ function getCliArgs(options = CLIOPTIONS, inputArgs = process.argv) {
  * @return {object}         - An object with errors and a boolean check
  */
 function checkCliInput(cliArgs, options = CLIOPTIONS) {
+	const result = {
+		pass: true,
+		errors: [],
+	};
+
 	const argDict = {};
-	Object.entries(options).map(([name, value]) => {
-		argDict[camelCase(name)] = name;
+	Object.entries(options).map(([key, value]) => {
+		argDict[camelCase(key)] = options[key];
 	});
 
-	Object.entries(cliArgs).map(([name, value]) => {
-		// console.log(name);
-		if (options[name]) {
-			// check type
-			// check arguments
+	Object.entries(cliArgs).map(([key, value]) => {
+		if (options[key]) {
```

Comment:
I'd add a debug call here:
```suggestion
		D.log(`Checking "${color.yellow(key)}"`);
		if (options[key]) {
```